pkg/context/cas/ariesstore/ariesstore.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package ariesstore
```

Comment:
I think this package should be called local (or something like that). We don't really care about the storage implementation - just that the CAS is stored locally.

Reply:
Currently all stores are implemented under /pkg/store so we can put this one in pkg/store/cas
Provider can be implemented in context/cas/??? and ipfs moved to context/cas/ipfs - let's discuss this.

Reply:
Looking through the spec, I'm actually not sure what the relation between the concept of `context` and `cas` is. Does it really belong under there? I might be missing something, though. If they are separate concepts, how about the following folder structure?

One funny thing about the term "local" is that actually you can easily pass in a distributed, shared storage provider (like CouchDB) into `ariesstore.New`, so "local" may be misleading. It's only "local" if you configured it that way.

Reply:
(Updated the folder structure based on our off-GitHub discussion)